README.md

```diff
@@ -47,31 +47,31 @@ Whenever an aliased PCA command is executed, it will use the current working dir
 | NO!      | `pca-import --customer ../CUST.json`     | parameter file is in a parent directory    |
 | NO!      | `pca-import --customer /tmp/CUST.json`   | parameter file is an absolute path         |
 
-### Advanced Configuration
+### Advanced configuration
 
-By default, the container will look for your PCA configuration in `/etc/pca`. This location can be changed by setting the `PCA_CONF_DIR` environment variable to point to your PCA configuration directory. The commands will also attempt to run using the `ncats/pca-core` image. A different image can be used by setting the `PCA_CORE_IMAGE` environment variable to the image name.
+By default, the container will look for your PCA configuration in `/etc/pca`. This location can be changed by setting the `PCA_CONF_DIR` environment variable to point to your PCA configuration directory. The commands will also attempt to run using the `cisagov/pca-core` image. A different image can be used by setting the `PCA_CORE_IMAGE` environment variable to the image name.
 
 Example:
 
 ```
 export PCA_CONF_DIR=/private/etc/pca
-export PCA_CORE_IMAGE=dhub.ncats.cyber.dhs.gov:5001/pca-core
```

Comment:
Antediluvian!

Reply:
Yeah- really getting rid of some old sauce here! 🌊